netcdf-java-bom/build.gradle.kts

```diff
@@ -0,0 +1,27 @@
+plugins {
```

Comment:
whats diff between  netcdf-java-bom and  netcdf-java-platform ?

Reply:
The netcdf-java-bom lists the artifacts that make up the netcdf-java library, along with their version.

https://artifacts.unidata.ucar.edu/repository/unidata-snapshots/edu/ucar/netcdf-java-bom/8.0.0-SNAPSHOT/netcdf-java-bom-8.0.0-20211229.052136-33.pom

It gives users a one-stop-shop for accessing netcdf-java artifacts (like the grpc bom above).

The netcdf-java-platform lists all of the direct dependencies used by the artifacts listed in netcdf-java-bom. This is useful for something like the TDS, in which we want to make sure the versions of, say, protobuf, are aligned with what netCDF-Java uses.

https://artifacts.unidata.ucar.edu/repository/unidata-snapshots/edu/ucar/netcdf-java-platform/8.0.0-SNAPSHOT/netcdf-java-platform-8.0.0-20211229.052136-33.pom

I cleaned this up quite a bit, so the version in the link above includes some things that don't really matter to our users (like the bounce dependency that part of toolsUI).

Reply:
Also, we dogfood the netcdf-java-platform (you will see it at the top of the dependency list for every project), which ensures that each project uses the exact same versions of the core dependencies.